deploy/openshift/clusterrole-monitoring-patch.yaml

```diff
@@ -0,0 +1,11 @@
+- op: add
```

Comment:
We don't need this patch by the way, the default SCC set by OpenShift (`restricted`) is fine.

Reply:
Removed